mantidproject/mantid
# Muon plot clean up

This PR cleans up the muon plotting code base. It removes the unused PlotMode and the unused files. 

I have also added in a check to prevent errors when using both FDA (Frequency Domain Analysis) and MA (Muon Analysis) at the same time.



**To test:**


Test 1:
 Play with MA and FDA (including fitting)

Test 2:
Load a run into MA
With MA open, load the same run into FDA
Should not get an error


Fixes #31497. 



It is not in the release notes as it is the final stage of the plotting refactor and is not user facing.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
